domains/emissions/Emissions.API/API/Helpers/Configuration.cs

```diff
@@ -7,6 +7,8 @@ public static class Configuration
         private const string RenewableSources = "RENEWABLESOURCES";
         private const string WasteRenewableShare = "WASTERENEWABLESHARE";
 
+        public static int DecimalPrecision => 5;
```

Comment:
why => 
?

Reply:
Short hand for a property with just a getter and a default value.